src/core/election.ts

```diff
@@ -86,7 +84,7 @@ export abstract class ElectionCore extends TransactionCore {
           blockCount: endBlock - (election.startDate ? startBlock : actualBlock),
           censusRoot: Uint8Array.from(Buffer.from(election.census.censusId, 'hex')),
           censusURI: election.census.censusURI,
-          status: ElectionStatus.READY,
+          status: ProcessStatus.READY,
```

Comment:
@marcvelmer if in the preparedElction data you set the election to Ready directly what is the point of th `continueEleciton` function in the client? For the paused ones? (just asking, not implying sth)